packages/dds/sequence/src/sequence.ts

```diff
@@ -566,12 +566,6 @@ export abstract class SharedSegmentSequence<T extends ISegment>
     }
 
     protected registerCore() {
-        for (const value of this.intervalMapKernel.values()) {
-            if (SharedObject.is(value)) {
-                value.bindToContext();
-            }
-        }
-
         this.client.startOrUpdateCollaboration(this.runtime.clientId);
```

Comment:
sequence calls this under a few conditions, connect, register, and attach. i think attach should be enough if things work how i think they do. i worked on some tests over the break that never went in, but they would provide some decent coverage here if sequence was also added:
https://github.com/microsoft/FluidFramework/pull/8377

Reply:
the above PR never went it, but if you approve it i can merge it

Reply:
I'm guessing that should be sufficient and there's nothing different that matrix is doing with startOrUpdateCollaboration which would justify having separate coverage for that?

Reply:
yeah. it's just not immediately clear we call connect or attach is all the places covered by register. if one dds works, they all should. the test is in. also, make sure you modify the dds (shared string) in all its states, specifically after create, and after attach